build-farm/platform-specific-configurations/linux.sh

```diff
@@ -82,7 +87,21 @@ then
     fi
     export JDK_BOOT_DIR=$JDK10_BOOT_DIR
 fi
-if [ "${JAVA_TO_BUILD}" == "${JDK11_VERSION}" ] || [ "${JAVA_TO_BUILD}" == "${JDKHEAD_VERSION}" ] || [ "${VARIANT}" == "${BUILD_VARIANT_OPENJ9}" ]; then
+
+if [ "${JAVA_TO_BUILD}" == "${JDK12_VERSION}" ] || [ "${JAVA_TO_BUILD}" == "${JDKHEAD_VERSION}" ]
+then
+    if [ ! -d "$JDK11_BOOT_DIR" ]; then
+      export JDK11_BOOT_DIR="$PWD/jdk-11"
+      if [ ! -d "$JDK11_BOOT_DIR/bin" ]; then
+        downloadArch="${ARCHITECTURE}"
+        [ "$downloadArch" == "arm" ] && downloadArch="arm32"
+        mkdir -p "$JDK11_BOOT_DIR"
+        wget -q -O - "https://api.adoptopenjdk.net/v2/binary/nightly/openjdk11?os=linux&release=latest&arch=${downloadArch}&type=jdk&openjdk_impl=openj9&heap_size=normal" | tar xpzf - --strip-components=1 -C "$JDK11_BOOT_DIR"
```

Comment:
again hotspot by default?

Reply:
Again I don't mind as long as a unique build is returned.  I'll change the non-aix ones to hotspot pending a resolution regarding aix hotspot builds not being returned by the api.

Reply:
Changed to hotspot